packages/graph/hash_graph/lib/graph/src/lib.rs

```diff
@@ -41,9 +41,9 @@
 #![allow(clippy::use_self, reason = "Too many false positives")]
 
 pub mod api;
-pub mod datastore;
 pub mod logging;
-pub mod types;
+pub mod ontology;
```

Comment:
Do you want to add an empty `knowledge` module as well?

Reply:
I thought about it but I'm not sure how we feel about empty modules. Maybe it's best to help convey the intended trajectory

Reply:
Yes, that was my thought as well. I also try to avoid empty modules, but we expect the module to be implemented in the next two days 🙂

Reply:
I have added